package.json

```diff
@@ -9,6 +9,11 @@
     "type": "git",
     "url": "https://github.com/RentPath/jquery-autotagging.git"
   },
+  "peerDependencies": {
```

Comment:
So, `peerDependencies` isn't deprecated, per se.  Its behavior is changed to warn that the parent project is not including the correct package version (or the entire package).  Version 3 no longer auto-installs peer dependencies.

I switched from `dependencies` to `peerDependencies` based on a comment @aribouius made [here](https://github.com/rentpath/jquery-moving-service.js/pull/24).

However, looking through the [version 3 changelog](https://github.com/npm/npm/releases/tag/v3.0.0) and reading up on the "flat" dependencies section, it looks like all `dependencies` get installed exactly once *except* in the case where two or more packages are in conflict.

There are pros and cons to both.  If you use `peerDependencies`, then you get the warning if you try using a package without the correct dependencies installed (good).  However, now it's up to the parent project to keep track of all package sub-dependencies (not so good). I can imagine dependencies can accumulate in the parent project's `package.json` over time and now you have to manually keep track of which sub-dependencies are no longer used.

With `dependencies`, you do not need to worry about sub-dependencies.  The parent project only cares about the libraries it immediately needs, and npm will auto-install everything else.  With npm v3, if all sub-dependencies are the same version then no problem.  However, if one library uses say jQuery 1.3 and another uses jQuery 1.9, then you may end up with both versions installed and code passing around two incompatible jQuery objects.  That's my understanding, anyway.

I'm leaning towards `peerDependencies`, just because of the possibility of multiple versions silently conflicting.  That, and the fact that we probably don't want multiple versions of a package installed (even if the code using them does not conflict) simply to cut down on the size.  Multiple versions makes sense in node, but not so much for the web.  Currently we get around this with bower's resolutions.  We resolve, for example, Flight to v1.5.0 despite other libraries using 1.3 or whatever.  Unfortunately, `peerDependencies` is not an exact replacement for bower resolutions.  It's strict, which means you *have* to satisfy the version in the parent project or go and fix the version in the library.  Where bower says "ignore the versions that libraries request and just go with version X", npm says "the installed version must satisfy all library requirements."  That can be tricky, so npm recommends loose versioning...

> Trying to install another plugin with a conflicting requirement will cause an error. For this reason, make sure your plugin requirement is as broad as possible, and not to lock it down to specific patch versions.

Any thoughts?